motr/ut/idx_dix.c

```diff
@@ -446,7 +446,8 @@ static uint64_t dix_val(uint64_t i)
 	return 100 + i * i;
 }
 
-static void ut_dix_record_ops(bool dist, uint32_t flags)
+static void ut_dix_record_ops(bool dist, uint32_t cr_get_flags,
+			      uint32_t put_del_flags)
```

Comment:
What's the point of having different args for the flags?

Reply:
To reuse the code as much as possible.
These flags are for entity/dix operations. 
cr_get_flags is used when doing entity creation and dix get.
put_del_flags is used when doing dix put and dix del.

So, we can do a sequence of operations, with different flags.

Reply:
I see. Thanks.